src/datasource/precomputed/annotations.md

```diff
@@ -109,6 +109,7 @@ Within the annotation id index, each annotation is encoded in the following bina
   - For `"LINE"` type, the first endpoint position followed by the second endpoint position.
   - For `"AXIS_ALIGNED_BOUNDING_BOX"` type, the first position followed by the second position.
   - For `"ELLIPSOID"` type, the center position followed by the radii vector.
+  - For `"POLYLINE"` type, the number of points as a uint32le value, followed by the position vectors of each point as float32le.
```

Comment:
can you also update docs/datasource/precomputed/annotation.rst

Reply:
Hi @jbms, I updated the `src/datasource/precomputed/annotation.rst` to account for a fifth annotation type which seems to be symlinked from the `docs/datasource/precomputed/annotation.rst`. Let me know if I'm missing the intent there and I'll take another look, and sorry for the delay on this